Design HomeScreen

rushiii3 opened the pull request:
#### PR Description
Please include a summary of the changes and the related issue. Describe your changes in detail.

#### Type of Change
- [ ] Bug fix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Documentation update

#### Select your work-area
- [x] Frontend
- [ ] Backend
- [ ] Documentation
- [ ] Others

#### Related Issue
Please provide a link to the issue solved if applicable.

#### Add your Work Example
<video src="https://github.com/SB2318/UltimateHealth/assets/105168088/113c0786-511b-408f-963b-16fe7eed0124" controls > </video>
<video src="https://github.com/SB2318/UltimateHealth/assets/105168088/39a7fc9c-882a-4747-9c84-2f4211ad5bb0" controls > </video>



#### Fixes (mention the issue number which this fixes)
closes #113 

#### Checklist
- [x] I have updated my branch and synced it with the project's 'develop' branch before making this PR.
- [x] I have optimized the file changes.
- [x] I have added a snapshot of my work example.
- [x] I have made a PR to the project's develop branch.

#### Undertaking
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] I have checked for plagiarism and assure its authenticity.
- [x] I have read and followed the code of conduct for this repository. I understand that violation of this undertaking may have legal consequences.

- [x] I Agree

rushiii3 commented:
Do not merge!
I am having a merge conflict that I need to resolve. I will remerge after resolving the conflict.

SB2318 reviewed (approved):
Nice Work! 🎉

Please increase the font-size of tab elements and increase fontFamily as well (bold)

Do same for the description Text below 'Discover' (But keep it little smaller.)


![WhatsApp Image 2024-06-18 at 1 05 35 AM](https://github.com/SB2318/UltimateHealth/assets/87614560/08c29b77-cee7-42bc-8eec-1d3f5d806e9f)



Update your branch.

rushiii3 commented:
Changed tabbar label size and fontweight bold 
Changed the font size of header subtitle to a lesser font size 
<img src="https://github.com/SB2318/UltimateHealth/assets/105168088/9fa6954d-fbdd-4bd4-83b3-43767dbd1350" height="600" />
